doc/rados/operations/balancer.rst

```diff
@@ -21,7 +21,7 @@ To check the current status of the balancer, run the following command:
 Automatic balancing
 -------------------
 
-When the balancer is in ``upmap`` mode, the automatic balancing feature is
+When the balancer is in ``upmap`` mode, the automatic upmap balancing feature is
 enabled by default. For more details, see :ref:`upmap`.  To disable the
```

Comment:
When the balancer is in the ``upmap`` mode, which is the default, the automatic upmap balancing feature is enabled.  For more ...

Reply:
Fixed